plugins/snippets/luasnip/default.nix

```diff
@@ -164,6 +185,11 @@ in
               ''
             ))
           ];
+
+      filetypeExtendConfig = mapAttrsToList (n: v: ''
+        require("luasnip").extend_filetypes("${n}", ${helpers.toLuaObject v})
```

Comment:
Just a thought, is there a way we could have an attrsOf `<fn_name, settings>` option? That way we support arbitrary functions in a freeform way:

```nix
require('luasnip').${name}(${toLuaObject value})
```

I guess this doesn't work well here because the function takes two arguments.

Reply:
That's a good idea indeed.